examples/monitoring/v1alpha1/scylladbmonitoring.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBMonitoring
+metadata:
+  name: test
+spec:
+  endpointsSelector:
+    matchLabels:
+      app.kubernetes.io/name: scylla
+      scylla/cluster: basic # Adjust this to match your ScyllaCluster name
```

Comment:
nit: comments above the referencing line

Reply:
removing the comment and embedding it into the value, so this can eventually be tool formatted (tools loose comments)